src/domain/Reward.ts

```diff
@@ -125,7 +122,6 @@ export function calculateRewards(date: BigInt, pool: Pool): void {
     }
     // if no linked address is found, we track reward in linkableRewards
     else if (rewardsAreClaimable(date, reward.nonZeroBalanceSince)) {
```

Comment:
can remove this else if block, as the condition is now covered in the below else

Reply:
Good find, removed it :)